sql/2021/http/response_number_and_size.sql

```diff
@@ -0,0 +1,32 @@
+# standardSQL
+# Number and Size of all responses by percentile
```

Comment:
Is this a duplicate of `pages_most_requests.sql`?

Reply:
As per above `pages_most_requests.sql` is the top 100 worst offenders for an interesting call out. We list the sites to make sure they are real things and not crawl errors.

This is the percentile version of that (as you were suggesting I use above!) more for graph.